uint8 id_task
int64 sec
int64 nsec
float32 deadline
float32 utility_max
uint8 total_ports
